### ISSUE TITLE
Please support recent versions of QuickCheck and Tasty

### ISSUE THREAD

**Issue by peti** [dmcclean/exact-pi#9]
I tried building this package in NixOS but failed because the build does not accept recent versions of the QuickCheck and Tasty packages. Would it be possible to remedy that?

http://packdeps.haskellers.com/feed?needle=exact:exact-pi

**Comment by Bodigrim** [comment]
I have published a new revision on Hackage. Does it work for you?

**Comment by peti** [comment]
Yes, that fixes the issue. Thank you very much for the quick response!
